corehq/toggles.py

```diff
@@ -1041,6 +1041,19 @@ def _commtrackify(domain_name, toggle_is_enabled):
     [NAMESPACE_DOMAIN]
 )
 
+LEGACY_CHILD_MODULES = StaticToggle(
```

Comment:
Should this and the migration be in a separate PR that's released before this one, so the legacy apps are flagged before the new code goes out?

Reply:
Sorry, I thought I'd replied to this days ago.  I wanted the migration be run during the deploy since it also reorders modules in some circumstances, which I'd like to happen right when we roll out the new UI, otherwise it might introduce additional confusion.

Reply:
sounds good